lib/types/addons.d.ts

```diff
@@ -0,0 +1,93 @@
+import API, { INormalizeError, IRazorpayQuery } from './api'
+import { IRazorpayItemId } from './items';
+
+
+export interface IRazorpayAddon {
+    /**
+     * A name for the add-on. For example, `Extra appala (papadum)`.
+     */
+    name?: string;
+    /**
+     * The amount you want to charge the customer for the add-on, in the currency subunit. For example, `30000`.
+     */
+    amount?: number;
+    /**
+     * The currency in which the customer should be charged for the add-on. For example, `INR`.
+     */
+    currency?: string;
+    /**
+     * Description for the add-on. For example, `1 extra oil fried appala with meals`
+     */
+    description?: string;
+}
+
+export interface IRazorpayAddonId {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Lets rename this to IRazorpayAddon

Reply:
Done